shared/locales/intl/unitformat.en-US.properties

```diff
@@ -1,79 +1,86 @@
+# Based on CLDR Unit Fields
+# http://www.unicode.org/cldr/charts/29/summary/en.html#5150
+
+# NOTICE: The {value} should *not* be localized. It's not a localizable
+# variable either. It's used by Intl to deconstruct the pattern.
+
 duration-second-narrow        = {[ plural(value) ]}
-duration-second-narrow[zero]  = {{ value }}s
-duration-second-narrow[one]   = {{ value }}s
-duration-second-narrow[two]   = {{ value }}s
-duration-second-narrow[few]   = {{ value }}s
-duration-second-narrow[many]  = {{ value }}s
-duration-second-narrow[other] = {{ value }}s
+duration-second-narrow[zero]  = {value}s
```

Comment:
@zbraniecki Do you have any plan for how to notify the localizers about this change?  This file already has translations in the repo, e.g. http://hg.mozilla.org/gaia-l10n/it/file/tip/shared/intl/unitformat.properties